pkgs/development/python-modules/warrant/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, buildPythonPackage, fetchFromGitHub, fetchPypi
+, boto3, envs, python-jose-cryptodome, requests }:
+
+buildPythonPackage rec {
+  pname = "warrant";
+  version = "0.6.1";
+
+  # move to fetchPyPi when https://github.com/capless/warrant/issues/97 is fixed
+  src = fetchFromGitHub {
+    owner  = "capless";
+    repo   = "warrant";
+    rev    = "ff2e4793d8479e770f2461ef7cbc0c15ee784395";
+    sha256 = "0gw3crg64p1zx3k5js0wh0x5bldgs7viy4g8hld9xbka8q0374hi";
+  };
+
+  propagatedBuildInputs = [ boto3 envs python-jose-cryptodome requests ];
```

Comment:
I opened https://github.com/capless/warrant/pull/98 for using python-jose instead.

Reply:
You should be able to use that as a patch, which would get rid of all problems.

Reply:
@peterhoeg Did you try applying that patch?

Reply:
No, haven't touched this yet. This weekend!